packages/postcss/mixin.core.js

```diff
@@ -1,10 +1,6 @@
-// XXX: This file creates multiple side-effects
-require('./patch-postcss-modules-extract-imports');
```

Comment:
Doesn't seem to be necessary anymore, the resolution of the imports in `spec/integration/postcss` works fine without it.